packages/test/test-service-load/src/utils.ts

```diff
@@ -96,7 +96,8 @@ export async function initialize(testDriver: ITestDriver, seed: number) {
         urlResolver: testDriver.createUrlResolver(),
         documentServiceFactory: testDriver.createDocumentServiceFactory(),
         codeLoader: createCodeLoader(random.pick(randEng, generateRuntimeOptions(seed))),
-        logger: ChildLogger.create(await loggerP, undefined, {all: { driverType: testDriver.type }}),
+        logger: ChildLogger.create(await loggerP, undefined, {all: { driverType: testDriver.type,
+                hostName: "Stress Tests" }}),
```

Comment:
can you actually incorporate this into the logger created in loggerP

Reply:
Do you mean like adding an event property in FileLogger's send method?

Reply:
yes